spec/rails_helper.rb

```diff
@@ -24,6 +24,7 @@ def conn
   config.include(MyAppHelper)
 
   config.before(:suite) do
+    `rake db:migrate`
```

Comment:
This is my stop-gap solution to provide the same functionality as `ActiveRecord::Migration.maintain_test_schema!` by the way